karyon3-core/src/main/java/com/netflix/karyon/http/ServerConfig.java

```diff
@@ -1,9 +1,9 @@
-package com.netflix.karyon.rxnetty;
+package com.netflix.karyon.http;
 
 import com.netflix.archaius.annotations.Configuration;
 import com.netflix.archaius.annotations.DefaultValue;
 
-@Configuration(prefix="karyon.rxnetty")
+@Configuration(prefix="karyon.httpserver")
```

Comment:
Agreed.  I'll move to a server package.  There will eventually be the notion of a client.